openedx/core/lib/hash_utils.py

```diff
@@ -32,3 +35,32 @@ def short_token():
     Currently, this is just meant to create sufficiently random tokens
     """
     return create_hash256(max_length=32)
+
+
+def hash_usage_key(usage_key: UsageKey) -> str:
+    """
+    Get the blake2b hash key for the given usage_key and encode the value. The
+    hash key will be added to the usage key's mapping dictionary for decoding
+    in LMS.
+
+    Args:
+        usage_key: the id of the location to which to generate the path
+
+    Returns:
+        The string of the encoded hashed key.
+    """
+    short_key = hashlib.blake2b(bytes(str(usage_key), 'utf-8'), digest_size=6)
+    encoded_hash = urlsafe_b64encode(bytes(short_key.hexdigest(), 'utf-8'))
+    return str(encoded_hash, 'utf-8')
```

Comment:
@KristinAoki This is pulled near-verbatim from your implementation in `openedx.core.djangoapps.course_experience.url_helpers`. The differences are:
* It's in a more generic location now. I didn't want to make `learning_sequences` depend on something course-specific like `course_experience`.
* It does not save the mappings to a persisted in-process dictionary; it's a pure function now.

Reply:
I ended up moving this again; it's in `openedx.core.djangoapps.content.learning_sequences.api.key_hashing` now.